apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -118,72 +176,180 @@ export default class ExportHTMLModal extends Vue {
   // If we are exporting a profile we can remove the test/results table
   @Prop({type: String, required: true}) readonly fileType!: string;
 
+  // Generated injectable HTML for icons
+  private iconHTMLStore = {
+    // Passed; green
+    circleCheck: this.iconDatatoSVG(
+      mdiCheckCircle,
+      'rgb(76, 176, 79)',
+      'A green circle with a check'
```

Comment:
make sure the descriptions actually have meaning associated with them not just the physical description.  as in, green circle with a check is not that useful when the actual meaning is that this was a successful control or whatever.  maybe we don't even need to have a description at all since the picture is fully supplementary to what's going on and there's always a text label that explains in full what's happening.  i would look up documentation on how to write good alt text - maybe ask on #accessibility or #uiux.

Reply:
Changed as requested.